README.md

```diff
@@ -161,6 +161,11 @@ Then clean and build project using **./gradlew clean build**
    rm -rf bitcoin-0.18.0*
    As a validation, you can run *bitcoind -daemon*. Run *bitcoin-cli stop* afterwards.
 ```
+
+**Important:**
+When setting up the Bitcoind Node host:port for the powpeg-node you MUST use the p2p port. 
+The powpeg-node will “connect” even if you setup the RPC port, but the connection won’t go through and the powpeg-node won’t start.
+
```

Comment:
This should go below the sample configuration file that is above.

There should also change 
```
# peers for the bitcoin network
    bitcoinPeerAddresses = [
        "127.0.0.1:18332" #bitcoind host and port, change if running on a different port.
    ]
```
for 
```
# peers for the bitcoin network
    bitcoinPeerAddresses = [
        "127.0.0.1:18444" #bitcoind p2p port
    ]
```

Reply:
Added bitcoind p2p port